proposals/cli-jar-processing.md

```diff
@@ -0,0 +1,48 @@
+# Proposal : Jib CLI Jar Processing 
+
+Relevant Issue: [#2796](https://github.com/GoogleContainerTools/jib/issues/2796)
+
+# Motivation 
+Allow users to containerize arbitrary jar files without having to integrate Jib into java build tools (maven and gradle) or use a `.yaml` jib cli buildfile.
+
+# Proposal Changes
+
+## Standard Jar 
+A standard jar can be containerized in two modes, exploded or packaged. 
+
+### Exploded Mode
+Achieved by calling `jib jar ${JAR_NAME}.jar --to ${TARGET_REGISTRY}`
+The default mode for containerizing a jar. It will explode a jar into the following layers:  
+- Dependencies Layer: Contains dependencies derived from `Class-Path` in jar manifest. Note that this layer will not be created if `Class-Path` is not present in the manifest.
+- Resources Layer: Contains resources parsed from jar file. Note that it will also include `MANIFEST.MF`.
+- Classes Layer: Contains classes parsed from jar file. 
+
+**Entrypoint** : `java -cp /app/dependencies/ /app/explodedJar/ ${MAIN_CLASS}`
```

Comment:
```suggestion
**Entrypoint** : `java -cp /app/dependencies/:/app/explodedJar/ ${MAIN_CLASS}`
```
Would be `;` instead of `:` on Windows, but we don't support creating Windows images anyway.